### ISSUE TITLE
Image urls don't appear to be escaped when in an array in some cases - Multiple backgrounds

### ISSUE THREAD

**Issue by Frannsoft** [timhagn/gatsby-background-image#138]
## Description

I have a client with an image url being used for a background and the image url has a set of parentheses in the name (e.g., road(1).jpg). The component that renders this uses multiple backgrounds, one of them being the supplied url and the other being a `linear-gradient`. 

I'm querying for this as a `fluid` image with webp:
```
file(relativePath: { eq: "road(1).jpg" }) {
    childImageSharp {
      # Specify the image processing specifications right in the query.
      # Makes it trivial to update as your page's design changes.
      fluid(maxWidth:2000) {
        ...GatsbyImageSharpFluid_withWebp
      }
    }
```

It seems like `gatsby-background-image` isn't applying single quotes to the url of the background image in this scenario because the background image isn't being rendered and upon inspecting the `background-image` css property for the component in devtools Chrome is warning me that it is invalid.

Example invalid `background-image` according to Chrome:
`background-image: linear-gradient(rgba(255,255,255,0.85), rgba(255,255,255,0.85)),url(http://localhost:8000/static/ecabadd464c99a6375eac44f6fb909d9/25f09/road(1).webp);`

### Steps to reproduce

Query for a fluid image
Apply multiple backgrounds to a `BackgroundImage` component by sending in an array of backgrounds
- In my example the first is `linear-gradient`
- The second is the fluid image object

Here is a demo repo: https://github.com/Frannsoft/gbi-url-demo 

### Expected result

The `url` in the `background-image` should have single quotes applied. E.g., `url('https://mysite.com/myimg(1).webp')`

### Actual result

The `url` in the `background-image` css does not have single quotes applied, making it invalid. E.g., `url(https://mysite.com/myimg(1).webp)`

### Environment

```
  System:
    OS: Windows 10 10.0.18363
    CPU: (4) x64 Intel(R) Core(TM) i5-4690K CPU @ 3.50GHz
  Binaries:
    Node: 12.13.1 - C:\Program Files\nodejs\node.EXE
    npm: 6.12.1 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 2.7.16 - /c/Python27/python
  npmPackages:
    gatsby: ^2.25.3 => 2.25.3 
    gatsby-background-image: ^1.2.0 => 1.2.0 
    gatsby-image: ^2.5.0 => 2.5.0 
    gatsby-plugin-sharp: ^2.8.0 => 2.8.0 
    gatsby-source-filesystem: ^2.5.0 => 2.5.0 
    gatsby-transformer-sharp: ^2.6.0 => 2.6.0 
  npmGlobalPackages:
    gatsby-cli: 2.12.60
    gatsby: 2.19.43
```

Thank you for working on this library! Please let me know if you need any other info or something isn't clear.

**Comment by timhagn** [comment]
Hi @Frannsoft,

Thanks for the thorough & (nearly ; ) complete issue! Gonna try looking into it today.
Only one thing: which browser have to tested it on?

Best,

Tim.

**Comment by Frannsoft** [comment]
Hi @timhagn,

So far I've tested it on Chrome 86.0.4240.183 and Firefox 82.0.2.

**Comment by timhagn** [comment]
Hi again @Frannsoft!

Thnx for commenting so quickly with your browsers (was afraid I'd have to install Edge ; ).
Looking into it I found out that in `getUrlString()` of `ImageUtils.js` called `hasImageUrl()` with the image array & not with the `currentString` in its loop % ). One more bug down ^^. At least it now works as intended with your example repo at my place with the just published `gbi@1.3.1`. Should it work for you as well, you may of course have the honor of closing this issue : ).

Best,

Tim.

P.S.: Nice Image of a German (speaking) town ; ).

**Comment by Frannsoft** [comment]
This seems to fix the issue on my client's site as well. Thank you!
